src/cdk-experimental/listbox/listbox.ts

```diff
@@ -67,6 +68,9 @@ export class CdkOption implements ListKeyManagerOption, Highlightable {
     this._disabled = coerceBooleanProperty(value);
   }
 
+  /** The form value of the option. */
+  @Input() value: any;
```

Comment:
Maybe the option should be generic so people can type the value if they want to? E.g.

```
class CdkOption<T = any> {
  value: T;
}
```

Reply:
Hmm this seems like a good idea. Do you know if there's a reason why MatOption doesn't do this?

Reply:
+1, `MatOption` didn't do this because we didn't know any better back when we wrote it. I would, though, use `unknown` instead of `any` for the default